Sources/TuistSupport/Constants.swift

```diff
@@ -18,6 +18,7 @@ public enum Constants {
     public static let masterKey = "master.key"
     public static let encryptedExtension = "encrypted"
     public static let templatesDirectoryName: String = "Templates"
+    public static let resourceTemplatesDirectoryName: String = "ResourceTemplates"
```

Comment:
Nitpick: What about calling this one `ResourceSynthesizers`?

Reply:
yeah, shorter while keeping the same explicitness, I like it 👍